src/components/TeacherComponents/OnboardingForm/onboardingActions.ts

```diff
@@ -74,3 +81,32 @@ export async function getSubscriptionStatus(
     });
   }
 }
+
+/**
+ * Sends onboarding data to hubspot
+ */
+export async function onboardUserToHubspot(data: OnboardingHubspotFormData) {
+  const hubspotFormId = getBrowserConfig("hubspotOnboardingFormId");
+  const hubspotFormPayload = getHubspotOnboardingFormPayload({
+    hutk: getHubspotUserToken(),
+    data,
+  });
+
+  try {
+    await hubspotSubmitForm({
+      hubspotFormId,
+      payload: hubspotFormPayload,
+    });
+  } catch (error) {
+    if (error instanceof OakError) {
+      reportError(error);
+    } else {
+      reportError(
+        new OakError({
+          code: "hubspot/unknown",
+          originalError: error,
+        }),
+      );
+    }
+  }
+}
```

Comment:
I lifted this out of onboarding form in its entirety just to make things more readable.